src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -1204,10 +1204,17 @@ def _need_update_container(self):
 
     def set_up_existing_container_update(self):
         if self.get_argument_force_single_container_updates():
+            containers = safe_get(self.new_containerapp, "properties", "template", "containers", default=[])
+            if len(containers) == 0:
+                raise ValidationError(
+                    "Cloud build image update failed. The container app '{}' does not have any containers.".format(self.get_argument_name()))
             # Remove n-1 containers where n is the number of containers in the containerapp and replace remaining container with the new container
             # Fails if all containers are removed
             while (len(self.new_containerapp["properties"]["template"]["containers"]) > 1):
```

Comment:
Remove redundant parentheses for `while (len(self.new_containerapp["properties"]["template"]["containers"]) > 1):`

Reply:
It should be `while len(self.new_containerapp["properties"]["template"]["containers"]) > 1:`

Reply:
Updated.